bci_tester/data.py

```diff
@@ -203,13 +203,13 @@ def create_BCI(
 MINIMAL_CONTAINER = create_BCI(
     build_tag=f"bci/bci-minimal:{OS_VERSION}",
     image_type="kiwi",
-    available_versions=[OS_VERSION],
+    available_versions=["15.4"],
```

Comment:
@dcermak can you help me figure out why the CI complains with
```
==================================== ERRORS ====================================
_____________________ ERROR collecting tests/test_build.py _____________________
'bci/bci-minimal_15.3' not found in `markers` configuration option
```
since I'm forcing only 1 version?

Reply:
You are creating a `bci/bci-minimal:15.3` and `bci/bci-minimal:15.4` as the build tag comes from the `OS_VERSION` environment variable.

Reply:
So, it should be hardcoded also here:
`build_tag=f"bci/bci-minimal:15.4",`

We probably need to change this once 15.5 is there and we need to support 15.3 (bci-base only), 15.4 (all) and 15.5 (all)

Reply:
No, just don't remove the build tag from pyproject.toml. with `available_versions=["15.4"]` that container will be automatically skipped for 15 SP3